src/main.py

```diff
@@ -555,6 +555,7 @@ def postprocess(data, ctx):
         ('preprint', [jats('pub_history'), preprint_events, first, to_preprint, discard_if_none_or_empty]),
         ('received', [jats('history_date', date_type='received'), to_isoformat, discard_if_none_or_empty]),
         ('accepted', [jats('history_date', date_type='accepted'), to_isoformat, discard_if_none_or_empty]),
+        ('sent-for-peer-review', [jats('history_date', date_type='sent-for-review'), to_isoformat, discard_if_none_or_empty]),
```

Comment:
@gnott , do we have any articles with a `sent-for-review` `history_date` yet? I can't find any examples

Reply:
Confirmed with Fred, no there are no real published samples yet, two are in review though. The kitchen sink XML sample is the best illustration.

Reply:
Here's the first real data

https://api.elifesciences.org/articles/86628/versions

Reply:
`{"received": null, "accepted": null, "sentForReview": "2023-02-14", "versions": ...`

Should `received` and `accepted` be `null` like that? I'll take a look.

Reply:
logic is here: https://github.com/elifesciences/lax/blob/0055390974bcda298ece061a04d0ed7754ed0980/src/publisher/logic.py#L318-L339

but it could be this:

> the date received, scraped from the xml, is guaranteed to not exist for certain article types.

https://github.com/elifesciences/lax/blob/0055390974bcda298ece061a04d0ed7754ed0980/src/publisher/logic.py#LL319C7-L319C101

article type in question is a `tools-resources` article type. I remember when they were introduced and that they were a little different from regular articles.